active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -405,12 +405,79 @@ Provides event information when an app is launched.
 
 ### Properties
 
-- Arguments: Gets the arguments that are passed to the app during its launch.
+- Arguments: Gets the arguments that are passed to the app during its launch. In a UWP app, this is equivalent to LaunchActivatedEventArgs.Arguments.
     ```CS
     public String Arguments { get; };
     ```
-- LaunchActivatedEventArgs: Gets the LaunchActivatedEventArgs from UWP. This property gets Null in Win32.
+- LaunchActivatedEventArgs: Gets the LaunchActivatedEventArgs from UWP. This property is null in a Desktop app.
     ```CS
     public Windows.ApplicationModel.Activation.LaunchActivatedEventArgs UWPLaunchActivatedEventArgs { get; };
     ```
 
+# API Details
+
+## Window class
+Microsoft.UI.Xaml namespace
+
+```cs
+    [contract(Microsoft.UI.Xaml.WinUIContract, 1)]
+    [webhosthidden]
+    [contentproperty("Content")]
+    unsealed runtimeclass Window
+    {
+        ...
+
+        String Title;
+        ImageSource Icon;
+        Microsoft.System.DispatcherQueue DispatcherQueue{ get; };
+    };
+
+```
+
+## IWindowNative (COM) interface
+
+```cs
+[
+    object,
+    uuid( EECDBF0E-BAE9-4CB6-A68E-9598E1CB57BB ),
+    local,
+    pointer_default(unique)
+]
+interface IWindowNative: IUnknown
+{
+    [propget] HRESULT WindowHandle([out, retval] HWND* hWnd);
+};
+```
+
+## Application class
+Microsoft.UI.Xaml namespace
+
+```cs
+    [webhosthidden]
+    unsealed runtimeclass Application
+    {
+        ...
+
+        // This matches Windows.UI.Xaml.Application except for the new event args type
+        overridable void OnLaunched(XamlLaunchActivatedEventArgs args);
+    };
+```
+
+
+## Event args
+
+```cs
+[webhosthidden]
+runtimeclass XamlLaunchActivatedEventArgs
+{
+    String Arguments{ get; };
+    Windows.ApplicationModel.Activation.LaunchActivatedEventArgs UWPLaunchActivatedEventArgs{ get; };
+};
+```
+
+# Appendix
+
+Changes in this spec from Preview2:
+* Rename LaunchActivatedEventArgs to XamlLaunchActivatedEventArgs
```

Comment:
Isn't that already implied from the namespace? IMHO I find it a bit weird to prefix this with `Xaml` for an application that launched.

Reply:
That's to try and avoid confusion with the existing type named [LaunchActivatedEventArgs](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Activation.LaunchActivatedEventArgs) (in the Windows.ApplicationModel.Activation namespace). This Xaml type is a wrapper around that.

Another alternative could be to simply give this a different name; "launch activated" is a bit confusing in this context. For example `ApplicationActivatedEventArgs`.

Reply:
It seems that the preferred was XamlLaunchActivatedEventArgs. Resolving the conversation